src/thrift_0_13_0/Makefile

```diff
@@ -0,0 +1,30 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+THRIFT_VERSION = 0.13.0
+THRIFT_VERSION_FULL = $(THRIFT_VERSION)-6
+
+MAIN_TARGET = libthrift-$(THRIFT_VERSION)_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = libthrift-dev_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb \
+		  python3-thrift_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb \
+		  thrift-compiler_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+
+THRIFT_LINK_PRE = http://deb.debian.org/debian/pool/main/t/thrift/
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	rm -rf thrift-$(THRIFT_VERSION)
+
+	wget "$(THRIFT_LINK_PRE)/thrift_$(THRIFT_VERSION_FULL).debian.tar.xz"
+	wget "$(THRIFT_LINK_PRE)/thrift_$(THRIFT_VERSION).orig.tar.gz"
+	wget "$(THRIFT_LINK_PRE)/thrift_$(THRIFT_VERSION_FULL).dsc"
+
+	dpkg-source -x thrift_$(THRIFT_VERSION_FULL).dsc
+	pushd thrift-$(THRIFT_VERSION)
+
+	DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -d -rfakeroot -b -us -uc -j$(SONIC_CONFIG_MAKE_JOBS) --admindir $(SONIC_DPKG_ADMINDIR)
```

Comment:
>buildpackage [](http://example.com/codeflow?start=32&length=12)

Did you actually apply your patch before the build?

Reply:
Yep, missed few lines